test/base/EscrowFactoryCreateEscrowTest.js

```diff
@@ -42,10 +46,10 @@ contract('EscrowFactoryCreateEscrowTest', function (accounts) {
       }
       try {
         // Invocation
-        await instance.createEscrow(
-          borrower,
-          loanID,
-          { from: owner }
+        await loans.createEscrow(
```

Comment:
I think these tests are for EscrowFactory.createEscrow function (not loans)

Reply:
Indeed it is, however, we will need to add that check to the `EscrowFactory.createEscrow` function that the loans contract is the only one who can call this function. I updated it so it is using the `EtherCollateralLoansMock.externalCreateEscrow` to mock the call from it.